core-bundle/src/Resources/config/services.yml

```diff
@@ -502,17 +502,17 @@ services:
         tags:
             - routing.loader
 
-    contao.routing.input_enhancer:
+    contao.routing.enhancer.input:
```

Comment:
Again, I llike `contao.routing.input_enhancer` better.

Reply:
This name is now suggested automatically 🎉